pkg/cmd/extension/command.go

```diff
@@ -141,6 +141,40 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				return nil
 			},
 		},
+		&cobra.Command{
+			Use:   "create <name>",
+			Short: "Create a new extension",
+			Args:  cmdutil.ExactArgs(1, "must specify a name for the extension"),
+			RunE: func(cmd *cobra.Command, args []string) error {
+				extName := args[0]
+				if !strings.HasPrefix(extName, "gh-") {
+					extName = "gh-" + extName
+				}
+				if err := m.Create(extName); err != nil {
+					return err
+				}
+				if io.IsStdoutTTY() {
```

Comment:
Nit: you could invert the condition here and return early to avoid having to indent the rest of the function from here on